.travis.yml

```diff
@@ -36,8 +36,8 @@ before_install:
 - if [[ "$PLATFORM" =~ android ]]; then echo y | android update sdk -u --filter android-25,android-26;
   fi
 - git clone https://github.com/apache/cordova-paramedic /tmp/paramedic && pushd /tmp/paramedic
-  && npm install && popd
-- npm install -g cordova
+  && npm install && echo '{"platform":"ios","action":"run","cleanUpAfterRun":true,"verbose":true,"saucePlatformVersion":"10.0","sauceAppiumVersion":"1.6.4"}' > conf/pr/ios-11.0.config.json && popd
```

Comment:
@NiklasMerz This line is no longer needed. I only added it when I was testing `-env: PLATFORM=ios-11.0`

Reply:
Thanks